tools/doc/html.js

```diff
@@ -23,51 +23,71 @@
 
 const common = require('./common.js');
 const fs = require('fs');
-const marked = require('marked');
+const unified = require('unified');
+const find = require('unist-util-find');
+const visit = require('unist-util-visit');
+const markdown = require('remark-parse');
+const remark2rehype = require('remark-rehype');
+const raw = require('rehype-raw');
+const html = require('rehype-stringify');
 const path = require('path');
 const typeParser = require('./type-parser.js');
 
 module.exports = toHTML;
 
-// Make `marked` to not automatically insert id attributes in headings.
-const renderer = new marked.Renderer();
-renderer.heading = (text, level) => `<h${level}>${text}</h${level}>\n`;
-marked.setOptions({ renderer });
-
 const docPath = path.resolve(__dirname, '..', '..', 'doc');
 
+// Add class attributes to index navigation links.
+function navClasses() {
+  return walker;
+
+  function walker(node, file) {
```

Comment:
As we do not use the `file` argument, maybe we can drop it here and in the recursive calls?